Shared/Interpreters/HSceneInterpreter.cs

```diff
@@ -421,10 +421,17 @@ internal int GetCurrentLoop(bool increase)
         
         internal static void OnPoseChange(HSceneProc.AnimationListInfo anim)
         {
-            mode = anim.mode switch
+            //mode = anim.mode switch
+            //{
+            //    EMode.houshi or EMode.houshi3P or EMode.houshi3PMMF => EMode.houshi,
+            //    EMode.sonyu or EMode.sonyu3P or EMode.sonyu3PMMF => EMode.sonyu,
+            //    _ => anim.mode,
+            //};
+
+            mode = (int)anim.mode switch
             {
-                EMode.houshi or EMode.houshi3P or EMode.houshi3PMMF => EMode.houshi,
-                EMode.sonyu or EMode.sonyu3P or EMode.sonyu3PMMF => EMode.sonyu,
+                1 or 6 or 8 => EMode.houshi,
+                2 or 7 or 9 => EMode.sonyu,
```

Comment:
enums/constants can be used as they are even if they don't exist in party since they are compiled down into an int.

Reply:
```suggestion
                EMode.houshi or EMode.houshi3P or EMode.houshi3PMMF => EMode.houshi,
                EMode.sonyu or EMode.sonyu3P or EMode.sonyu3PMMF => EMode.sonyu,
```